accounts/sso/azure/index.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Single Sign-On: Microsoft Entra ID (Azure AD) integration'
+backlink_href: /docs/accounts/sso
+backlink_title: 'Single Sign-On'
+---
+
+pganalyze supports integrating with [Microsoft Entra ID (formerly Azure AD)](https://www.microsoft.com/en-us/security/business/identity-access/microsoft-entra-id)
+for Single Sign-On, using [SAML 2.0](https://learn.microsoft.com/en-us/entra/architecture/auth-saml).
+
+There are two steps for integrating with Microsoft Entra ID:
+
+1. Create a new Microsoft Entra ID Enterprise application
+2. Register the Microsoft Entra ID Enterprise application with pganalyze
+
+## Create a new Microsoft Entra ID Enterprise application
+
+In your Azure portal, navigate to **Enterprise applications** page, then select **New application**.
+On the New application page, select **Create your own application** to create an application for SSO.
+
+![Screenshot of Microsoft Azure portal showing Create your own application page](azure_create_enterprise_app.png)
+
+Once the application is created, select **Set up single sign on** panel.
+
+![Screenshot of Enterprise application page](azure_enterprise_app.png)
+
+In the setting up single sign-on page, choose **SAML** as a single sign-on method.
+
+![Screenshot of Setting up single sign on](azure_select_sso_method.png)
+
+In the step 1, **Basic SAML Configuration**, fill the **Identifier (Entity ID)** and the **Reply URL (Assertion Consumer Service URL)**.
+
+![Screenshot of Basic SAML Configuration](azure_setup_saml_step1.png)
+
+These can be found in the SAML settings page in pganalyze.
+To retrieve the SAML settings specific to your organization, you can navigate to the **Integrations**
+settings page in your pganalyze account, where you will see the **Single Sign-On with SAML** panel.
+
+![Screenshot of pganalyze SAML integration details showing ACS and Metadata URL](../sso_details_for_idp.png)
+
+The **Single Sign-On URL (ACS)** above is the one you'll need to copy to the **Reply URL (Assertion Consumer Service URL)**.
+
+Next, navigate to the step 2 and click **Edit**:
+
+![Screenshot of Attributes & Claims](azure_setup_saml_step2.png)
+
+In the Attributes & Claims page, click **Add new claim** to add a new claim.
+As a claim `email` is required, create a new claim with a name `email`. Please note that the namespace field _must be empty_.
+For the source attribute, choose `user.email`. If your organization keeps the user's email in the other attribute, pick that one.
```

Comment:
I feel like this wording is a little unclear. How about de-emphasizing that second case (I assume it's fairly rare?) with something like
```suggestion
For the source attribute, you'll typically choose `user.email` unless your organization keeps the user's email in another attribute.
```